app/commands/codemod/review.go

```diff
@@ -208,11 +208,15 @@ l:
 
 			if cfm.Confirm(0, iss, confirmSRC) {
 
+				issuePos := iss.Position
+				startPos := issuePos.GetStart()
+
 				// TODO(waigani) offsets need to be based off codemod decorator, not review.
```

Comment:
Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.